challenges/02-javascript-algorithms-and-data-structures/debugging-cn.json

```diff
@@ -0,0 +1,646 @@
+        }
+      ],
+      "solutions": [],
+      "hints": [],
+      "releasedOn": "Feb 17, 2017",
+      "challengeType": 1,
+      "translations": {},
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": [
+            "let x = 7;",
+            "let y = 9;",
+            "let result = \"to come\";",
+            "",
+            "if(x = y) {",
+            "  result = \"Equal!\";",
+            "} else {",
+            "  result = \"Not equal!\";",
+            "}",
+            "",
+            "console.log(result);"
+          ],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "id": "587d7b85367417b2b2512b39",
+      "title":
+        "Catch Missing Open and Closing Parenthesis After a Function Call",
+      "description": [
+        "当函数或方法不接受任何参数时，您可能忘记在调用它时加上（）左括号和右括号。通常，函数调用的结果会保存在变量中，以供代码中的其他用途使用。可以通过将变量值（或其类型）打印到控制台，会看到是一个函数引用而不是函数调用的返回值来检测此错误。",
+        "下面示例中的两个变量是不同的:",
+        "<blockquote>function myFunction() {<br>&nbsp;&nbsp;return \"You rock!\";<br>}<br>let varOne = myFunction; // set to equal a function<br>let varTwo = myFunction(); // set to equal the string \"You rock!\"</blockquote>",
+        "<hr>",
+        "修复代码，使变量 <code>result</code> 设置为调用函数 <code>getNin​​e</code> 返回的值。"
+      ],
+      "tests": [
+        {
+          "text":
+            "Your code should fix the variable <code>result</code> so it is set to the number that the function <code>getNine</code> returns.",
+          "testString":
+            "assert(result == 9, 'Your code should fix the variable <code>result</code> so it is set to the number that the function <code>getNine</code> returns.');"
+        },
+        {
+          "text": "Your code should call the <code>getNine</code> function.",
+          "testString":
+            "assert(code.match(/getNine\\(\\)/g).length == 2, 'Your code should call the <code>getNine</code> function.');"
+        }
+      ],
+      "solutions": [],
+      "hints": [],
+      "releasedOn": "Feb 17, 2017",
+      "challengeType": 1,
+      "translations": {},
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": [
+            "function getNine() {",
+            "  let x = 6;",
+            "  let y = 3;",
+            "  return x + y;",
+            "}",
+            "",
+            "let result = getNine;",
+            "console.log(result);"
+          ],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "id": "587d7b85367417b2b2512b3a",
+      "title":
+        "Catch Arguments Passed in the Wrong Order When Calling a Function",
+      "description": [
+        "继续讨论调用函数，需要注意的下一个 bug 是函数的参数传递顺序错误。 如果参数分别是不同的类型，例如接受数组和整数两个参数的函数，可能会引发运行时错误。 确保以正确的顺序提供所有必需的参数以避免这些问题。",
```

Comment:
`If the arguments are different types, such as a function expecting an array and an integer, this will likely throw a runtime error. If the arguments are the same type (all integers, for example), then the logic of the code won't make sense. `

貌似你漏了后面的一句。

`如果参数分别是不同的类型，例如接受数组和整数这两个参数的函数，参数顺序传错就可能会导致运行错误。对于接受相同类型参数的函数，传错参数也会导致逻辑错误或运行结果错误`